services/general-service/.env.example

```
@@ -1,34 +1,41 @@
+
+# Redis Configuration (Cache)
+REDIS_HOST=localhost
+REDIS_PORT=6379
+REDIS_PASSWORD=
+REDIS_DB=0
+
+# AWS Configuration (LocalStack)
+AWS_REGION=ap-southeast-1
+S3_BUCKET_URL=http://localhost:4566/fuvekonse-bucket
+SQS_QUEUE_URL=http://sqs.ap-southeast-1.localhost:4566/000000000000/fuvekon-queue
+SES_EMAIL_IDENTITY=fuve.vietnam@gmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SES_EMAIL_IDENTITY key should go before the SQS_QUEUE_URL key

(UnorderedKey)